openfe/setup/methods/rbfe_methods.py

```diff
@@ -190,7 +192,34 @@ class Config:
 
 class LigandLigandTransformResults:
     """Dict-like container for the output of a LigandLigandTransform"""
-    pass
+    def __init__(self, settings: LigandLigandTransformSettings):
+        self._parent_settings = settings
+        fn = self._parent_settings.simulation_length.output_filename
+        self._reporter = multistate.MultiStateReporter(fn)
+        self._analyzer = multistate.MultiStateSamplerAnalyzer(self._reporter)
+
+    def dG(self):
+        """Free energy difference of this transformation
+
+        Returns
+        -------
+        dG : unit.Quantity
+          The free energy difference between the first and last states. This is
+          a Quantity defined with units.
+        """
+        dG, _ = self._analyzer.get_free_energy()
```

Comment:
I _think_ this also works for SAMS but I might have to check later on, adding TODO for now.